src/index.ts

```diff
@@ -924,7 +929,7 @@ export class ValidationService {
      * @returns
      */
     private isHidden(input: HTMLElement) {
-        return !( input.offsetWidth || input.offsetHeight || input.getClientRects().length );
+        return !(this.allowHiddenFields || input.offsetWidth || input.offsetHeight || input.getClientRects().length );
```

Comment:
Does this mean by default, we don’t validate fields that are hidden because they’ve scrolled offscreen?

Reply:
Ah, looks like `offsetWidth` has nothing to do if it’s currently visible in the viewport.

Reply:
Yeah, I left the original logic, I'm guessing it knows how to figure it out